Refacto voucher part with boostrap components

NeOMakinG opened the pull request:
<!-----------------------------------------------------------------------------
Thank you for contributing to the PrestaShop project! 
Please take the time to edit the "Answers" rows below with the necessary information.
Check out our contribution guidelines to find out how to complete it:
https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requests
------------------------------------------------------------------------------>

## How does it looks?
![image](https://user-images.githubusercontent.com/14963751/158403594-29a4782d-fc08-40d4-9758-e21397888635.png)

@Hlavtox @mparvazi if you can review, would be appreciated!

<!-- Click the form's "Preview" button to make sure the table is functional in GitHub. Thank you! -->

mparvazi commented:
Need to add some help comment:
`If you have a voucher code paste here or click on voucher code in the voucher list.`

NeOMakinG commented:
> Need to add some help comment: `If you have a voucher code paste here or click on voucher code in the voucher list.`
> 
> Some customers have problem with this UX.

Can we talk about this on https://github.com/PrestaShop/theme-refacto/issues/51 ?

I think even your help comment is not clear, we should talk about it

NeOMakinG commented:
@SharakPL @mparvazi I've addressed your feedbacks, it should be good now

![image](https://user-images.githubusercontent.com/14963751/158561519-86bda4f6-d868-49a7-9d0f-1d08383bce5f.png)

wdyt?

SharakPL commented:
Great! Thanks

Hlavtox commented on templates/checkout/_partials/cart-voucher.tpl:
```diff
@@ -34,7 +34,7 @@
                 <span class="cart-voucher__name col">{$voucher.name}</span>
                 <div class="d-flex align-items-center justify-content-end col">
-                  <span class="me-2">{$voucher.reduction_formatted}</span>
+                  <span class="me-2 fw-bold">{$voucher.reduction_formatted}</span>
```
@NeOMakinG Maybe ms-2 on the other element? If it's not deletable, there will be space.

Hlavtox commented on templates/checkout/_partials/cart-voucher.tpl:
```diff
@@ -34,7 +34,7 @@
               <li class="cart-voucher__item row">
                 <span class="cart-voucher__name col">{$voucher.name}</span>
                 <div class="d-flex align-items-center justify-content-end col">
```
col without row?

Hlavtox commented on templates/checkout/_partials/cart-voucher.tpl:
```diff
@@ -79,11 +79,11 @@
           {l s='Take advantage of our exclusive offers:' d='Shop.Theme.Actions'}
         </p>
-        <ul class="js-discount card-voucher__list">
+        <ul class="js-discount cart-voucher__list">
```
Do we want the same selector here? What about cart-voucher__offers?

Hlavtox commented on templates/checkout/_partials/cart-voucher.tpl:
```diff
@@ -79,11 +79,11 @@
+        <ul class="js-discount cart-voucher__list">
           {foreach from=$cart.discounts item=discount}
             <li class="cart-voucher__code">
               <span class="label">
```
Extra span

Hlavtox commented on templates/checkout/_partials/cart-voucher.tpl:
```diff
@@ -34,7 +34,7 @@
                 <div class="d-flex align-items-center justify-content-end col">
-                  <span class="me-2">{$voucher.reduction_formatted}</span>
+                  <span class="me-2 fw-bold">{$voucher.reduction_formatted}</span>
                     {if isset($voucher.code) && $voucher.code !== ''}
```
We should change this in the core. There should be a property `deletable` or something.

NeOMakinG replied:
Yeah but not related to hummingbird

Hlavtox commented:
Some small comments, otherwise rock and roll 🚀

I agree with @SharakPL and @mparvazi - everything must be made for dumbest people you can imagine.

NeOMakinG commented:
@Hlavtox Addressed all of your feedbacks

Looks mergeable now!

NeOMakinG commented:
Thanks @Hlavtox @SharakPL @mparvazi for the review, if you find something else later, don't hesitate to contribute or open an issue ❤️
